arch/cpu/cc26xx-cc13xx/rf-core/rf-core.c

```diff
@@ -577,16 +716,6 @@ cc26xx_rf_cpe1_isr(void)
       return;
     }
   }
-    
-  if(HWREG(RFC_DBELL_NONBUF_BASE + RFC_DBELL_O_RFCPEIFG) & IRQ_RX_BUF_FULL) {
-    PRINTF("\nRF: BUF_FULL\n\n");
-    /* set a flag that the buffer is full*/
-    rx_is_full = true;
-    /* make sure read_frame() will be called to make space in RX buffer */
-    process_poll(&rf_core_process);
-    /* Clear the IRQ_RX_BUF_FULL interrupt flag by writing zero to bit */
-    HWREG(RFC_DBELL_NONBUF_BASE + RFC_DBELL_O_RFCPEIFG) = ~(IRQ_RX_BUF_FULL);
-  }
```

Comment:
I am not 100% comfortable with the removal of this block. This was introduced to fix a very old and annoying prop-mode bug, see #567 and related issues. Are you 100% certain that this pull does not re-introduce the bug in CSMA operation?

Reply:
Thanks for catching this George - this is clearly removed accidentally, because my base code was older than this PR.